turbo.json

```diff
@@ -1,5 +1,19 @@
 {
   "$schema": "./docs/public/schema.json",
+  // The following are used by tools such as pnpm, gh actions to setup pnpm, and the Go toolchain
+  "globalPassThroughEnv": [
+    "Path",
+    "APPDATA",
+    "LOCALAPPDATA",
+    "TMPDIR",
+    "HOME",
+    "TMP",
+    "GOROOT",
+    "USERPROFILE"
+  ],
+  // This is probably overkill, but we do things per-platform often enough that we should segment
+  // everything by OS
+  "globalEnv": ["OS"],
```

Comment:
did you mean`RUNNER_OS` here? We have that set in some tasks also)
https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

Reply:
Hmm. I meant OS, since some tools read that, but we might actually want both. That will segment the cache between "runs on GH Actions" and running anywhere else.